typings/globals/he/index.d.ts

```diff
@@ -0,0 +1,112 @@
+// Generated by typings
```

Comment:
(FYI no action needed but most of this typings stuff will change or go away once we adopt the new typings system in TypeScript 2.0.)